docs/api/tools/block-explorer/contract-verification.md

```diff
@@ -37,49 +23,25 @@ To get started click the **Tools** tab in the top header, a dropdown will pop up
-- Optimization: This checks whether or not optimization was used while compiling the contract. If you enabled optimization during compilation, check **Yes** otherwise select **No**
-- Solidity compiler version(Solc): This specifies the exact version of the compiler used to compile a smart contract. Click on the dropdown to specify the compiler version used. We encourage you to use version `0.8.16`.
-- zkSync compiler version(Zksolc): The zkSync compiler version used, the default is set to `v1.2.0`.
-
-<br>
-
-![Compiler version!](../../../assets/images/compiler-version.png "compiler version")
-
-- Enter the Solidity Contract code: Copy the code from your editor and paste it into the text area.
-  <br>
-  **Note:** You may need to flatten your Solidity code if it utilizes a library or inherits dependencies from another contract.
-  <br>
-  We recommend using [Hardhat flatten](https://medium.com/coinmonks/flattening-smart-contracts-using-hardhat-dffe7dbc7b3f), [Truffle flattener](https://github.com/NomicFoundation/truffle-flattener) or the [POA Solidity flattener](https://github.com/poanetwork/solidity-flattener).
+- **Contract Address**: Enter the address that matches the 0x address generated during contract creation.
+- **Compiler Type**: Specify if your contract is a single or multi-part solidity file.
+- **Contract Name**: Enter the exact name of the contract as provided in the contract code.
+- **Source Code**: Copy and paste the code from your editor into the text area.
 
-- Constructor arguments: If constructor arguments are required by the contract, you should add them here, but in [ABI hex-encoded form](https://solidity.readthedocs.io/en/develop/abi-spec.html).
+ ::: tip
+If your Solidity code uses a library or inherits dependencies from another contract, you may need to flatten it using tools such as [Hardhat flatten](https://medium.com/coinmonks/flattening-smart-contracts-using-hardhat-dffe7dbc7b3f), [Truffle flattener](https://github.com/NomicFoundation/truffle-flattener), or [POA Solidity flattener](https://github.com/poanetwork/solidity-flattener).
+ ::: 
 
-<br>
-
-### How can you obtain your constructor arguments?
-
-> **Note:** The easiest way to get the constructor argument data is to print it to the console while deploying.
-
-For example, if using our [tutorial](../../../dev/building-on-zksync/hello-world.md), you can find the line:
+- **Solidity Version**: Specify the version of Solidity used in your code.
+- **Optimization**: Select "Yes" if you enabled optimization during contract compilation, otherwise select "No".
+- **Constructor Arguments**: If your contract has constructor arguments, provide them here.
+- **Solidity Compiler Version (Solc)**: Select the exact version of the compiler used to compile your smart contract from the dropdown.
+- **zkSync Compiler Version (Zksolc)**: Select `1.3.8` as the version of the zkSync compiler used during compilation.
```

Comment:
Can you link to the changelog so we don't have to update multiple lines every time there's an update? https://github.com/matter-labs/compiler-solidity/blob/main/CHANGELOG.md